cpp/src/phonenumbers/shortnumberinfo.h

```diff
@@ -162,6 +162,18 @@ class ShortNumberInfo {
       const PhoneNumber& number,
       const string& region_dialing_from) const;
 
+  // Given a valid short number, determines whether it is an SMS service
+  // (however, nothing is implied about its validity). An SMS service is where
+  // the primary or only intended usage is to receive and/or send text messages
+  // (SMSs). This includes MMS as MMS numbers downgrade to SMS if the other
+  // party isn't MMS-capable. If it is important that the number is valid, then
+  // its validity must first be checked using {@link #isValidShortNumber} or
+  // {@link #isValidShortNumberForRegion}. Returns false if the number doesn't
```

Comment:
Done! Thanks for catching.
Yeah I missed it too because it was last-minute copying from Java. I had just made sure it made sense and voila!